docs/integrations/hosts-operating-systems/opentelemetry/linux-opentelemetry.md

```diff
@@ -0,0 +1,255 @@
+
+### Host Metrics - Disk
+
+The Host Metrics - Disk dashboard provides detailed information about on disk utilization and disk IO operations.You can drill down from this dashboard to the Process Metrics - Details dashboard by using the honeycombs or line charts in all the panels.
+
+Use this dashboard to:
+
+-   Identify hosts with high disk utilization and disk IO operations.
+-   Monitor abnormal spikes in read/write rates.
+-   Compare disk throughput across storage devices of a host.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Linux-OpenTelemetry/Host-Metrics-Disk.png')} alt="Host Metrics - Disk" />
+
+### Host Metrics - Memory
+
+The Host Metrics - Memory dashboard provides detailed information on host memory usage, memory distribution, and swap space utilization. You can drill down from this dashboard to the Process Metrics - Details dashboard by using the honeycombs or line charts in all the panels.
+
+Use this dashboard to:
+
+-   Identify hosts with high memory utilization.
+-   Examine memory distribution (free, buffered-cache, used, total) for a given host.
+-   Monitor abnormal spikes in memory and swap utilization.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Linux-OpenTelemetry/Host-Metrics-Memory.png')} alt="Host Metrics - Memory" />
+
+### Host Metrics - Network
+
+The Host Metrics - Network dashboard provides detailed information on host network errors, throughput, and packets sent and received.
+
+Use this dashboard to:
+
+-   Determine top hosts with network errors and dropped packets.
+-   Monitor abnormal spikes in incoming/outgoing packets and bytes sent and received.
+-   Use dashboard filters to compare throughput across the interface of a host.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Linux-OpenTelemetry/Host-Metrics-Network.png')} alt="Host Metrics - Network" />
+
+### Host Metrics - TCP
+
+The Host Metrics - TCP dashboard provides detailed information around inbound, outbound, open, and established TCP connections.
+
+Use this dashboard to:
+
+-   Identify abnormal spikes in inbound, outbound, open, or established connections.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Linux-OpenTelemetry/Host-Metrics-TCP.png')} alt="Host Metrics - TCP" />
+
+### Process Metrics - Details
+
+The Process Metrics - Details dashboard gives you a detailed view of key process related metrics such as CPU and memory utilization, disk read/write throughput, and major/minor page faults.
+
+Use this dashboard to:
+
+-   Determine the number of open file descriptors in all hosts. If the number of open file descriptors reaches the maximum file descriptor limits, it can cause IOException errors.
+-   Identify anomalies in CPU usage, memory usage, major/minor page faults and reads/writes over time.
+-   Troubleshoot memory leaks using the resident set memory trend chart.
```

Comment:
```suggestion
- Determine the number of open file descriptors in all hosts. If the number of open file descriptors reaches the maximum file descriptor limits, it can cause IOException errors.
- Identify anomalies in CPU usage, memory usage, major/minor page faults and reads/writes over time.
- Troubleshoot memory leaks using the resident set memory trend chart.
```